microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -0,0 +1,288 @@
+
+:::image type="complex" source="../../media/2021/07/ua-client-hints-in-network-conditions.msft.png" alt-text="Defining User-Agent Client Hints for a custom device in the Network conditions tool" lightbox="../../media/2021/07/ua-client-hints-in-network-conditions.msft.png":::
+   Defining User-Agent Client Hints for a custom device in the **Network conditions** tool
+:::image-end:::
+
+For more information about this feature, navigate to [Detecting Microsoft Edge from your website][DevToolsWebPlatformUserAgentGuidance].<!--no images?-->
+<!-- optional todo: compare the What's New screenshots vs the screenshots in these two incoming articles: -->
+<!-- todo: link when this PR is merged:
+For more information about this feature, navigate to [Emulate mobile devices in Microsoft Edge DevTools](../../../device-mode/index.md#set-user-agent-client-hints) and [Network Analysis reference](../../../network/reference.md#set-user-agent-client-hints). -->
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1174299][CR1174299].
+
+
+<!-- 8 -->
+## Screen readers now announce errors, warnings, and issues in toolbar and Console
+
+<!-- Title: Better support for errors, warnings, and issues with assistive technology -->
+<!-- Subtitle: Screen readers now correctly announce the number and the type of notification for errors, warnings, and issues in the DevTools toolbar. -->
+
+Previously, users of screen readers would only hear the number of errors, warnings, or issues announced in the DevTools toolbar.  The additional information of what type of notification was being announced wasn't included, such as "Error, Warning, or Issue". For example, if the DevTools reported 3 errors, screen readers would just announce "3".
+
+Now in Microsoft Edge version 93, screen readers correctly announce the number of errors, warnings, or issues that the DevTools have detected and the type of notification. For example, if the DevTools reported 3 errors, screen readers now announce "3 errors".  This fix has been applied to both the notifications in the DevTools toolbar and in the Console.
+
+:::image type="complex" source="../../media/2021/07/screen-reader-errors-warnings-issues.msft.png" alt-text="The errors, warnings, and issues UI in the toolbar and Console" lightbox="../../media/2021/07/screen-reader-errors-warnings-issues.msft.png":::
+   The errors, warnings, and issues UI in the toolbar and **Console**
+:::image-end:::
+
+<!-- It'd be good to have a video of this a11y fix where the text that the screen reader announces is displayed -->
+
+For information about debugging Console errors, navigate to [Debug errors reported in Console](../../../console/console-debug-javascript.md).  For information about issues found by DevTools and improvements you can make to a webpage, navigate to [Find and fix problems using the Issues tool](../../../issues/index.md).
```

Comment:
Can we combine this line with the next sentence so that we only have one paragraph?